SUPPORT.md

```diff
@@ -0,0 +1,75 @@
+# Support
+
+This page explains where/how to get help with Principia. Please read through the
+following guidelines.
+
+## Discord
+
+Questions regarding the usage of Principia are best directed to the [Discord
+channel for
+Principia](https://discord.com/channels/319857228905447436/480397772248580098).
+There are many knowledgeable users there who can help you with basic questions
+and provide guidance on getting started with the mod.  The Discord channel is
+*not* suitable for reporting bugs, however; a post like "when I do X it crashes"
+will likely not been seen by the developers, and therefore the underlying bug
+won’t be fixed.
+
+## GitHub Issues
+
+GitHub Issues should be used for reporting bugs (either crashes or
+misbehaviours) and requesting new features or improvements.  GitHub is *not*
+suitable for questions on how to use the mod ("how do I go to the Moon from Cape
+Canaveral?", for instance).  Please use Discord for those.
+
+### Crashes
+
+There are two kinds of crashes that may be caused by Principia: crashes of 
+Principia itself, or crashes of KSP or Unity.
+
+When reporting a crash, please indicate whether the crash is reproducible.
+There is no need to include reproduction steps.
+
+#### Principia Crashes
+
+These crashes may manifest themselves as a Windows dialog box (on Windows) or a 
+SIGABRT (on Linux).  Or the game may just stop.
+
+Go to `<KSP directory>\glog\Principia`, you should find a `FATAL` file with a
+name that matches the time of the crash, for instance
+`FATAL.20230920-223637.6656.log`. You should also find an `INFO` file created
+shortly before that, for instance `INFO.20230920-223615.6656.log`.  Take these
+two files (one is not enough), upload them on gist. Link the gist in your github
```

Comment:
We have been recommending that for a while, but gist isn’t very convenient for that, since these files get far too large to be handled by a tool meant for small source code snippets.
We should recommend attaching the files to the issue instead: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files

Reply:
It might also be a good idea, here and below, to explicitly say that the entire unredacted unabridged file should be posted, not snippets that you think are interesting.